src/ExceptionTypes.f90

```diff
@@ -0,0 +1,370 @@
+!++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
+!                          Futility Development Group                          !
+!                             All rights reserved.                             !
+!                                                                              !
+! Futility is a jointly-maintained, open-source project between the University !
+! of Michigan and Oak Ridge National Laboratory.  The copyright and license    !
+! can be found in LICENSE.txt in the head directory of this repository.        !
+!++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
+!> @brief Utility module for defining exception types used by the
+!>  ExceptionHandler
+!>
+!> @author William Gurecky
+!>   @date 12/16/2019
+!++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
+MODULE ExceptionTypes
```

Comment:
All lines between `MODULE` and `ENDMODULE` need to be unindented by 2 spaces.  The style you did matches the old way of doing it, not the current code standard.  Much of Futility needs to be updated still to match the code standard (something I'd like to do soon).